### PR TITLE
[docs] several improvements

### PR DESCRIPTION
**Pull request by nqb**
# Description
Several improvements in docs

# Impacts
Docs

# Delete branch after merge
YES

### CODE REVIEW

**Comment by nqb** [comment]
Ready for review.

File: docs/PacketFence_Clustering_Guide.asciidoc [hunk 1]
```diff
@@ -1157,6 +1159,7 @@ At the end it you want to check the status of the slave server for debug purpose
 
  SHOW SLAVE STATUS;
 
+HAProxy daemon is responsible to detect when link between DC2 and DC1 is down.
```
Comment (julsemaan) [inline docs/PacketFence_Clustering_Guide.asciidoc]: Its a been random to put this here without any other context around it

Comment (nqb) [reply docs/PacketFence_Clustering_Guide.asciidoc]: A complete reorganization of this section is necessary to have something clear.

I prefer to have information even if it's not perfect.

Comment (julsemaan) [reply docs/PacketFence_Clustering_Guide.asciidoc]: At least make this clearer by saying its haproxy-db and talking explicitly about the database.

This sentence just looks like it was randomly dropped into the document and just looks odd.
Kinda like if @lzammit would go to a yoga class.

Comment (nqb) [reply docs/PacketFence_Clustering_Guide.asciidoc]: Is it better like this ?

**Pull request merged**